### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by Developers-codz**
Added responsiveness, absolute path and fixed dark mode

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ import {
   SinglePost,
 } from "./pages";
 import { ToastContainer, toast } from "react-toastify";
-import { getAllPosts } from "./Redux/Reducers/postsSlice";
-import { getAllUsers } from "./Redux/Reducers/userSlice";
-import { verifyToken } from "./Redux/Reducers/authSlice";
+import { getAllPosts } from "Redux/Reducers/postsSlice";
```
Comment (HeyNitin) [inline src/App.js:22]: Good use of absolute imports

**Review by HeyNitin — APPROVED** [review]
The code looks good, all good practices are there. UI is also clean and resposive, you can go ahead and merge

File: src/components/bottomNavigation/BottomNavigation.jsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+import { getBgColor, getTextColor,getBorder } from "../../utils/Functions/getColor";
 import { useTheme } from "../../context/theme-context";
 import { NavLink } from "react-router-dom";
 
 import styled from "styled-components";
 
 const Wrapper = styled.div`
   display: none;
-  @media only screen and (max-width: 560px) {
+  @media only screen and (max-width: 768px) {
```
Comment (arjun-computer-geek) [inline src/components/bottomNavigation/BottomNavigation.jsx:11]: suggestion : you can put multiple screen sizes. for multiple devices

**Review by arjun-computer-geek — APPROVED** [review]
Else are good. coding style is awesome.

File: src/App.js (hunk 1 above)
Comment (Developers-codz) [reply src/App.js:22]: Thank you

**Pull request merged**